### PR TITLE
Adding arch install instructions and removal of gtkmm3 dependencies

### PR DESCRIPTION
**Pull request by zalox**
* Add arch install instuctions
* Remove gtkmm3 as dependency (it gets automatically installed by gtksourceviewmm)

Before merge
- [x] Test instructions in arch @zalox
- [x] Test whether removal of gtkmm3 breaks install on other platforms
- [x] Add gtkmm3 to dependencies of gtksourceviewmm in msys2

### CODE REVIEW

**Comment by eidheim** [comment]
Great, nice to have arch linux instructions as well (although we should maybe ask them to update their gtksourceview*3 packages at some point, if they have not already done this). Do you have this platform available?

**Comment by zalox** [comment]
I'm leaving Debian on my laptop soon. Until then I have an friend who is going to test on arch.

**Comment by eidheim** [comment]
libgtkmm3.0 removal works on OS X, Debian testing/stable, Ubuntu 14
Does not work yet on MSYS2 (https://github.com/Alexpux/MINGW-packages/blob/master/mingw-w64-gtksourceviewmm3/PKGBUILD is missing gtkmm3 dependency, we should create a pull request).

**Comment by zalox** [comment]
I created https://github.com/Alexpux/MINGW-packages/pull/890.

**Comment by eidheim** [comment]
Splendid

**Comment by eidheim** [comment]
Since you are the Arch Linux guy, would you mind reporting https://www.archlinux.org/packages/community/x86_64/gtksourceviewmm/ out of date? Its still 3.2! While https://www.archlinux.org/packages/extra/i686/gtksourceview3/ is 3.18. Tell them that gtksourceviewmm should be called gtksourceviewmm3 as well.

**Comment by VadimGit** [comment]
What wrong with gtksourceviewmm in arch? Jucipp builds and runs on arch without any issues (related to gtkmm at least).

**Comment by eidheim** [comment]
gtksourceviewmm(3) has been flagged outdated, hopefully they will upgrade it soon.

**Comment by eidheim** [comment]
https://www.archlinux.org/packages/community/x86_64/gtksourceviewmm/ is now 3.18.

File: docs/install.md [hunk 1]
```diff
@@ -65,6 +65,22 @@ make
 make install
 ```
 
+##Arch
+Install dependencies:
+```sh
+pacman -S git cmake make clang gtksourceviewmm3 boost aspell aspell-en
+```
```
Comment (zalox) [inline docs/install.md]: Note: omit the 3.

File: docs/install.md [hunk 2]
```diff
@@ -65,6 +65,22 @@ make
 make install
 ```
 
+##Arch
+Install dependencies:
+```sh
```
Comment (zalox) [inline docs/install.md]: as root

**Comment by zalox** [comment]
@eidheim Please review latest changes and merge :)

File: docs/install.md (hunk 2 above)
Comment (eidheim) [inline docs/install.md]: not sudo on Arch, like on Debian and Ubuntu 14?

Comment (zalox) [inline docs/install.md]: You can install sudo in arch aswell, but some users like to su root.

Comment (eidheim) [inline docs/install.md]: sudo is safer to use in general, and you seldom see `su root` in installation documents. sudo is most commonly used I think.

**Pull request merged**